src/integrations/sveltekit/config.ts

```diff
@@ -0,0 +1,102 @@
+import type { ResolvedConfig } from 'vite'
+import type { ManifestTransform } from 'workbox-build'
+import type { BasePartial, GlobPartial, RequiredGlobDirectoryPartial } from 'workbox-build/src/types'
+import type { SvelteKitVitePluginOptions, VitePWAOptions } from '../../types'
+
+// Vite generates <name>.<hash>.<ext> layout while SvelteKit generates <name>-<hash>.<ext>
+// Vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
+
+interface WorkboxConfig extends BasePartial, GlobPartial, RequiredGlobDirectoryPartial {}
+
+// All assets will go to the immutable folder, and so, there is no need to calculate its revision for the sw's precache manifest
+export function configureSvelteKitOptions(viteOptions: ResolvedConfig, options: Partial<VitePWAOptions>) {
+  const {
+    base = viteOptions.build.base ?? '/',
+    adapterFallback,
+  } = options.svelteKitVitePluginOptions ?? {}
+
+  if (typeof options.includeManifest === 'undefined')
+    options.includeManifest = 'client-build'
+
+  // SvelteKit will copy static folder content to `.svelte-kit/output/client`.
```

Comment:
this should be the default because Vite will always copy `publicDir` to `outDir` and isn't anything specific to SvelteKit
```suggestion
  // Vite will copy static folder content to `.svelte-kit/output/client`.
```

Reply:
Kit will copy to the globDirectory before pwa plugin runs: globDirectory is the kit client output folder
Vite will copy public folder to the globDirectory after pwa plugin runs: globDirectory is the build folder

Reply:
ok, I believe you the plugin default can't be updated. still I think this comment should be updated as it's Vite that copies the static folder content. There is no code in SvelteKit to do that

Reply:
Yeah, I need also update the docs.